### PR TITLE
DDF-{4365, 4370, 4372, 4374} Bugfixes for source and subscription management

### PR DESCRIPTION
**Pull request by SmithJosh**
##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
Link to 2.13.x PR: #4082
____
#### What does this PR do?
This is a forward port of some bugfixes related to source and subscription management. It also improves the output of the `subscriptions:list` command

#### Who is reviewing it? 
@bakejeyner @blen-desta @Kjames5269 

#### Select relevant component teams: 
@codice/security 
@codice/ui

#### How should this be tested?
TEST 1: Sources UI
1. Create a new source in the sources UI, and open the browser dev tools to view the network tab.
2. Select and close the source edit modal a few times, and disable/re-enable the source a few times.
3. Verify there is still just a single request to the backend every 30 seconds.
4. Disable the source. Verify that the requests to the backend stop

TEST 2: subscriptions:list AccessControlException
1. Create a subscription pointing to a site that DDF does not trust, e.g. example.com
2. Run the subscriptions:list command and verify that there is no AccessControlException

#### Ask 2 committers to review/merge the PR and tag them here.

@coyotesqrl 
@brjeter 

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews.

### CODE REVIEW

**Comment by SmithJosh** [comment]
build now

File: ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Status.js [hunk 1]
```diff
@@ -17,13 +17,9 @@ define(['backbone'], function(Backbone) {
   var Status = {}
 
   Status.Model = Backbone.Model.extend({
-    url:
+    urlRoot:
       '../jolokia/exec/org.codice.ddf.catalog.admin.poller.AdminPollerServiceBean:service=admin-source-poller-service/sourceStatus/',
     model: Status.Model,
-
-    initialize: function(pid) {
```
Comment (Kjames5269) [inline ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Status.js]: Just curious, what does removing this do?

Comment (SmithJosh) [reply ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Status.js]: I needed a way to update the pid on the model. Using [urlRoot](http://backbonejs.org/#Model-urlRoot) allows this and was cleaner than refactoring to add a `setPid` method.

**Comment by Kjames5269** [comment]
Hero'd ✅

**Pull request merged**